python/pyabacus/README.md

```diff
@@ -9,9 +9,10 @@ Python).
 Installation
 ------------
 
-- install pybind11 and scikit-build-core by `pip install pybind11 scikit-build-core`
-- clone this repository
-- `pip install ./pyabacus`
+- Create and activate a new conda env, e.g. `conda create -n myenv python=3.8 & conda activate myenv`.
```

Comment:
```suggestion
- Create and activate a new conda env, e.g. `conda create -n pyabacus && conda activate pyabacus`.
```

Reply:
Since our package named as `pyabacus`, I was worried that user may feel confused and they can install other packages for DFT computation in their conda env.

Reply:
```suggestion
- Create and activate a new conda environment, e.g. `conda create -n myenv && conda activate myenv`.
```